internal/controllers/helpers.go

```diff
@@ -44,3 +46,34 @@ func getPrivileges(c *fiber.Ctx) []privileges.Privilege {
 	}
 	return privs
 }
+
+const dateLayout1 = time.DateOnly
+const dateLayout2 = time.RFC3339
```

Comment:
Is this relabeling helping?

Did not know about `DateOnly` tbh.

Reply:
I think it's new?